frontend/src/app/testQueue/TestCard/TestCard.tsx

```diff
@@ -66,6 +65,29 @@ export const TestCard = ({
     testOrderId: testOrder.internalId,
   };
 
+  const trackTimerReset = () => {
+    if (appInsights) {
+      appInsights.trackEvent({ name: "Test timer reset" }, timerContext);
```

Comment:
This is copied from TestTimer.tsx since invoking `timer.reset()` on the timer requires passing a `trackClickEvent` callback function which also needs a reference to `appInsights`. Honestly not sure if these tracked events are still relevant for us, but for the present time I opted to leave it as is. Could possibly refactor these callbacks into a separate custom hook that also uses that appInsights hook, if we prefer making that change I'm open to adding that in this PR or in a follow up